packages/jest-config/src/readConfigFileAndSetRootDir.ts

```diff
@@ -124,6 +124,7 @@ async function registerTsNode(): Promise<Service> {
       moduleTypes: {
         '**': 'cjs',
       },
+      transpileOnly: process.env.DISABLE_TS_NODE == 'True',
```

Comment:
I don't think this is a good name. What about `JEST_CONFIG_TRANSPILE_ONLY`?

Reply:
sure 👍